_DeploymentAndDistroPackaging/ansible/cleanup/ciao.yml

```diff
@@ -17,16 +17,11 @@
   connection: local
   tasks:
     - name: Remove local files
-      local_action: file path={{ item }} state=absent
-      with_items:
-        - ./openrc
-        - ./admin-ciaorc
-        - ./demo-ciaorc
-        - ./clouds.yaml
-        - ./certificates
-        - ./images/.cnci_ready
-        - ./fetch
-        - /tmp/go
+      file: path={{ item }} state=absent
+      with_fileglob:
+        - /tmp/go/*
```

Comment:
This looks insecure.  If I'm on the deployment machine you're about to run on, and I place a rootkitted go in there ahead of you, you will inherit my damage.  Or?